### PR TITLE
add feature extraction code

### PR DESCRIPTION
**Pull request by schan27**

### CODE REVIEW

File: feature_extraction/expanded_word_list.txt [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+mother
+newborn
+only woman
+other child
+other woman
+own child
+own kid
+parent
+people under
+pre
```
Comment (schan27) [inline feature_extraction/expanded_word_list.txt:22]: Remove

File: feature_extraction/expanded_word_list.txt [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+minor
+mother
+newborn
+only woman
+other child
+other woman
+own child
+own kid
+parent
+people under
```
Comment (schan27) [inline feature_extraction/expanded_word_list.txt:21]: Unsympathetic

File: feature_extraction/expanded_word_list.txt [hunk 3]
```diff
@@ -0,0 +1,45 @@
+babies
+child
+children
+daughter
+father
+female
+infant
+just woman
+male
```
Comment (schan27) [inline feature_extraction/expanded_word_list.txt:9]: Unsympathetic

File: feature_extraction/expanded_word_list.txt [hunk 4]
```diff
@@ -0,0 +1,45 @@
+babies
+child
+children
+daughter
+father
+female
```
Comment (schan27) [inline feature_extraction/expanded_word_list.txt:6]: Unsympathetic

File: feature_extraction/expanded_word_list.txt [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+infant
+just woman
```
Comment (schan27) [inline feature_extraction/expanded_word_list.txt:8]: Remove

File: feature_extraction/expanded_word_list.txt [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+male
+many baby
+many child
+minor
```
Comment (schan27) [inline feature_extraction/expanded_word_list.txt:12]: Unsympathetic

File: feature_extraction/expanded_word_list.txt [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+newborn
+only woman
+other child
+other woman
+own child
+own kid
+parent
+people under
+pre
+preteen
```
Comment (schan27) [inline feature_extraction/expanded_word_list.txt:23]: Unsympathetic

File: feature_extraction/expanded_word_list.txt [hunk 8]
```diff
@@ -0,0 +1,45 @@
...
+single woman
+small boy
+small child
+step mother
+teen
```
Comment (schan27) [inline feature_extraction/expanded_word_list.txt:28]: Unsympathetic

File: feature_extraction/expanded_word_list.txt [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+teenager
```
Comment (schan27) [inline feature_extraction/expanded_word_list.txt:29]: Unsympathetic

File: feature_extraction/expanded_word_list.txt [hunk 10]
```diff
@@ -0,0 +1,45 @@
...
+toddler
+underage
```
Comment (schan27) [inline feature_extraction/expanded_word_list.txt:31]: Unsympathetic

File: feature_extraction/expanded_word_list.txt [hunk 11]
```diff
@@ -0,0 +1,45 @@
...
+underage kid
```
Comment (schan27) [inline feature_extraction/expanded_word_list.txt:32]: Unsympathetic

File: feature_extraction/expanded_word_list.txt [hunk 12]
```diff
@@ -0,0 +1,45 @@
...
+very young child
+woman
+women
+young boy
+young child
+young girl
+young lady
+young man
+young men
+young teen
```
Comment (schan27) [inline feature_extraction/expanded_word_list.txt:42]: Unsympathetic

File: feature_extraction/expanded_word_list.txt [hunk 13]
```diff
@@ -0,0 +1,45 @@
...
+young teenager
```
Comment (schan27) [inline feature_extraction/expanded_word_list.txt:43]: Unsympathetic

File: feature_extraction/expanded_word_list.txt [hunk 14]
```diff
@@ -0,0 +1,45 @@
...
+young woman
+youth
```
Comment (schan27) [inline feature_extraction/expanded_word_list.txt:45]: Unsympathetic

**Review by imteazs — APPROVED** [review]

**Pull request merged**